src/app/Rules/Rules.tsx

```diff
@@ -241,6 +266,14 @@ export const Rules = () => {
       sorted = direction === SortByDirection.asc ? sorted : sorted.reverse();
     }
     return sorted.map((r: Rule) => [
+      <>
+        <Switch
+          label=""
+          aria-label={`${r.name} is enabled`}
```

Comment:
I think this aria-label should be given `enabled/disabled` based on the `r.enabled` field?

Reply:
Hmm, maybe. The docs say this:

```
  /** Adds accessible text to the switch, and should describe the isChecked="true" state. When label is defined, aria-label should be set to the text string that is visible when isChecked is true. */
  'aria-label'?: string;
```

I'm not sure exactly how the `aria-label` ends up getting used but I figured it was something like this string is used as a key and the switch state is the corresponding value for the screen reader or other assistive tech.

Reply:
Right! Just reading the usage of the label here: https://www.patternfly.org/v4/components/switch/#without-label

Looks like its used just in `checked` state for accessibility.